dvc/stage.py

```diff
@@ -213,6 +213,13 @@ def is_import(self):
         """Whether the stage file was created with `dvc import`."""
         return not self.cmd and len(self.deps) == 1 and len(self.outs) == 1
 
+    @property
+    def is_pkg_import(self):
```

Comment:
if feels like an output can decide if it needs to be cached or not

Reply:
Output needs to know if he is a part of `import` stage, so this is still used.

Reply:
not necessarily, when we create an output we can pass a flag. It can be and probably should be decided on the Cmd/API level. I can imagine we will have a flag to cache output anyway.

Reply:
Will keep that in mind in https://github.com/iterative/dvc/issues/2138 (added a comment about this discussion there as well).